candidate-standard/clause_0_front_material.adoc

```diff
@@ -2,13 +2,11 @@
 
 [big]*i.     Abstract*
 
-The OGC has extended their suite of standards to include Resource Oriented Architectures and Web Application Programming Interfaces (APIs). These standards are based on a common foundation upon which all OGC APIs will be built. This document extends that foundation to define the Environmental Data Retrieval API.
+The Environmental Data Retrieval (EDR) Application Programming Interface (API) provides a family of lightweight interfaces to access Environmental Data resources by requesting data at a *Position*, within an *Area* or along a *Trajectory*. An Environmental Data resource is a collection of spatiotemporal data that can be sampled using the EDR query patterns. These patterns are described in the <<rc_core-section,Requirement Class Core>> section. 
 
-An Environmental Data Retrieval (EDR) API provides a family of lightweight interfaces to access Environmental Data resources by requesting data at a Position, within an Area or along a Trajectory. In the context of OGC API collections of geospatial data, an Environmental Data resource is a collection of spatiotemporal data that can be sampled using the EDR query patterns. Each resource addressed by an EDR API maps to a defined query pattern. These patterns are described in the <<rc_core-section,Requirement Class Core>> section. 
+EDR API query patterns, such as Position, Time-series, Area, Cube, Trajectory or Corridor, can be thought of as discrete sampling geometries, conceptually consistent with the feature of interest in the https://www.ogc.org/standards/sos[Sensor Observation Service (SOS)]. A typical EDR data resource is a multidimensional dataset that possibly could be accessed via the http://www.ogc.org/standards/wcs[Web Coverage Service (WCS)]. In contrast to SOS and WCS, EDR implements the technical baseline of the OGC API family of standards and aims to provide a simple-to-use, single set of user-centric query patterns. EDR use cases range from location-centric real or virtual observation retrieval to sub-setting 4-dimensional data cubes along user-supplied sampling geometries. These query patterns do not attempt to satisfy the full scope of either SOS or WCS, but provide useful building blocks to allow the composition of APIs that satisfy a wide range of geospatial data use cases.
```

Comment:
Time-series ?

Reply:
I was trying to be explicit about what can be retrieved, rather than leaving it implicitly in Position and Area. I'll think of phrasing that does not imply another keyword.